pkg/kubectl/cmd/cmd_test.go

```diff
@@ -161,7 +161,7 @@ func Example_printMultiContainersReplicationControllerWithWide() {
 		NegotiatedSerializer: ns,
 		Client:               nil,
 	}
-	cmd := NewCmdRun(tf, os.Stdin, os.Stdout, os.Stderr)
+	cmd := resource.NewCmdGet(tf, os.Stdout, os.Stderr)
```

Comment:
Drop all 4 tests, we don't guarantee output stability. Additionally, the server-side print is already tested as part of this test: https://github.com/kubernetes/kubernetes/blob/master/test/integration/apiserver/print_test.go